lib/matplotlib/cbook.py

```diff
@@ -708,6 +708,9 @@ def is_string_like(obj):
         return False
     return True
 
+def is_list_like(obj):
```

Comment:
There are some files which are not checked (so we didn't have to do a massive touch-everything PR to get the entire codebase pep8 compliant), I suspect cbook is one of them as we don't touch this file very often.